app/segments/drill/[id]/submission/input.js

```diff
@@ -455,6 +589,8 @@ export default function Input({ drillInfo, setToggleResult, setOutputData }) {
 
   const { id: did } = useLocalSearchParams();
 
+  const currentLeaderboard = useLeaderboard({ drillId: did });
```

Comment:
I understand why isLoading doesn't need to be handled here, but what about error? If isLoading and error is not handled, then just access "data" here instead of in function